### PR TITLE
[CI/Build] Add dots.tts weekly e2e coverage

### PR DESCRIPTION
**Pull request by linyueqian**
## Summary

dots.tts landed in #4765 without any GPU coverage. Nothing under `tests/e2e/` exercises it, so the DiT flow-matching sampler, the streaming AudioVAE decode and the patch-encoder AR loopback currently have no CI signal on real hardware.

This adds `tests/e2e/offline_inference/test_dots_tts_expansion.py` marked `slow` + `tts` + L4, which the existing weekly **TTS · L4** step already collects via `-m "slow and L4 and tts"`. No pipeline YAML change is needed: the weekly TTS steps are marker driven, unlike the per-model reliability steps.

## Cases

| Test | Guards |
|---|---|
| `test_dots_tts_zero_shot_001` | Synthesis produces finite, non-silent audio of a plausible duration. |
| `test_dots_tts_prefill_decode_mixed_batch_002` | `compute_logits` fills `logits[i]` positionally from `_results_queue`, so every request must push exactly one entry per step, prefills included. Guards the row pairing raised in review of #4765. |
| `test_dots_tts_deterministic_flow_matching_noise_003` | The DiT noise comes from a per-request `blake2b(seed:request_key:noise_step)` generator rather than the global CUDA RNG. Guards the seeding fix from the same review. |

Note that dots.tts cannot use a bare `{"prompt": "..."}` string like most TTS models here: prefill needs `[文本]<text>[文本对应语音]<|audio_gen_start|>` as real Qwen2 token ids, so the helper goes through `build_dots_tts_prompt`.

## Test Plan

Selector behaviour verified against the pipelines' own expressions:

```
-m "slow and L4 and tts"        ->  3 collected      (weekly TTS · L4)
-m "full_model and L4 and tts"  ->  0 collected      (nightly, intentionally out of scope)
```

`ruff check` and `ruff format` clean at the repo's pinned v0.14.10.

The three cases have **not** been executed on an L4 yet, since they need the weekly runner. Adding the `weekly-test` label to this PR will validate them on the right hardware before merge.

## Context

Related to this, #4765's CPU unit tests were also never collected: the file carried no markers, so the ready pipeline's `-m 'core_model and cpu'` deselected all five. That was fixed in `8209be4` on the same PR. This one closes the remaining GPU-side gap.

cc @yenuo26 @congw729

### CODE REVIEW

**Review by Sy0307 — APPROVED** [review]
LGTM
